File: stress/t/opt_spill_float.c
#include <stdio.h>
int main(void){
  double f0=1.5,f1=2.5,f2=3.5,f3=4.5,f4=5.5,f5=6.5,f6=7.5,f7=8.5,f8=9.5,f9=10.5,
    f10=11.5,f11=12.5,f12=13.5,f13=14.5,f14=15.5,f15=16.5,f16=17.5,f17=18.5,f18=19.5,f19=20.5,sum;
  sum=f0+f1+f2+f3+f4+f5+f6+f7+f8+f9+f10+f11+f12+f13+f14+f15+f16+f17+f18+f19;
  sum=sum+f0*f19+f1*f18+f9*f10;
  printf("sum=%f\n",sum);
  printf("f0=%f\n",f0);
  printf("f10=%f\n",f10);
  printf("f19=%f\n",f19);
  return 0;
}

File: stress/t/opt_spill_float.f
      PROGRAM OSPFLT
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      F0=1.5D0
      F1=2.5D0
      F2=3.5D0
      F3=4.5D0
      F4=5.5D0
      F5=6.5D0
      F6=7.5D0
      F7=8.5D0
      F8=9.5D0
      F9=10.5D0
      F10=11.5D0
      F11=12.5D0
      F12=13.5D0
      F13=14.5D0
      F14=15.5D0
      F15=16.5D0
      F16=17.5D0
      F17=18.5D0
      F18=19.5D0
      F19=20.5D0
      SUM=F0+F1+F2+F3+F4+F5+F6+F7+F8+F9+F10+F11+F12+F13+F14+F15+
     &  F16+F17+F18+F19
      SUM=SUM+F0*F19+F1*F18+F9*F10
      WRITE(*,'(A,F0.6)') 'sum=',SUM
      WRITE(*,'(A,F0.6)') 'f0=',F0
      WRITE(*,'(A,F0.6)') 'f10=',F10
      WRITE(*,'(A,F0.6)') 'f19=',F19
      END
